pages/open-source.tsx

```diff
@@ -317,50 +317,52 @@ OSS.pageId = PageId.OSS;
 export default OSS;
 
 export const getStaticProps: GetStaticProps = async () => {
-  const orgName = "thirdweb-dev";
-
+  // Array of accounts to be tracked
+  const accounts = ["thirdweb-dev", "thirdweb-example"];
   const authHeader = {
     headers: {
       Authorization: `token ${process.env.GITHUB_API_TOKEN}`,
     },
   };
 
-  // Fetch the list of all repositories belonging to the organization
-  const reposResponse = await fetch(
-    `https://api.github.com/orgs/${orgName}/repos?per_page=100`,
-    authHeader,
-  );
-
-  const reposData = (await reposResponse.json()) as GithubRepository[];
-
-  const repos = reposData
-    .filter((repo) => repo.fork === false)
-    .filter((repo) => repo.name !== "shopify-thirdweb-theme")
-    .map((repo) => repo.name);
-
   const contributors: Record<string, GithubContributor> = {};
 
-  for (const repo of repos) {
-    const response = await fetch(
-      `https://api.github.com/repos/${orgName}/${repo}/contributors`,
+  for (const account of accounts) {
```

Comment:
could do this simultaneously instead of one at a time (use promise all, merge the results and then process them all together)